Assets/Shaders/Outlined_Diffuse.shader

```diff
@@ -0,0 +1,78 @@
+﻿// Upgrade NOTE: replaced 'mul(UNITY_MATRIX_MVP,*)' with 'UnityObjectToClipPos(*)'
```

Comment:
This shader looks pretty cool.

Reply:
To be clear, I did not make this shader but works really well lol